person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_writer :id, :name, :age
```

Comment:
- [ ] @rahimialijan Great work on your class, however  `name` and `age` are supposed to have both reader and setter since we will be modifying the variables, so we should have  ` attr_accessor :name, :age ` and ``attr_reader :id``